DigitalOffice/Shared/Kernel/Broker/OperationResultWrapper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.Kernel.Broker
+{
+    /// <summary>
+    /// Wraps the returned object from a function in OperationResult.
+    /// </summary>
+    /// <typeparam name="T">parameter of the called function</typeparam>
+    /// <typeparam name="TRes">response from the called function, which will be wrapped in OperationResult</typeparam>
```

Comment:
Writing it wrong here because the class is not generic. You must write this next to the generic method.